analyses/selection-strategy-comparison/01-selection-strategies.rmd

```diff
@@ -38,9 +38,18 @@ set.seed(2019)
 
 ```
 
+### Environment
+
+Get the UNIX environment for CI options.
+```{r unix environment}
+ENV <- Sys.getenv()
+UMAPN <- as.numeric(ENV['UMAPN'])
```

Comment:
If you're reading this directly in R, the environment variable will be `OPENPBTA_UMAPN`. In my example the `OPENPBTA_` bit goes away because the bash script doesn't use it.

Reply:
Noted; I missed that line in the explanation and thought it might have been stripped in the transfer to Docker. Of course the fact that it ran confirms my suspicion that it wasn't needed for this notebook: it was failing for other reasons.

Reply:
If I knew how to do the direct R use of env variables with defaults, the docs would say that instead. It'd be cleaner - I just don't know enough R to give the example 😂 . Feel free to update the readme if you know how 😁

Reply:
In most cases, I would think your way is better. There was a bit of unclean logic in mine, because I was using a notebook, which generally wants to avoid talking to the environment, so you can't pass in variables directly. I think `Sys.getenv()` as above will work, but I will have to test it with something that actually uses the env variables.